tests/test_hello_world.py

```diff
@@ -93,17 +116,56 @@ def test_make_me_2_functions_one_use_fixture_and_one_use_parametrize():
         assert times_7(rnd_int) == sum([rnd_int for i in range(7)])
 
         # assert times_7(rnd_int) > rnd_int  # TODO Explain why this assert doest work
+        # this does not hold in case rnd_int is negative.
+        # also, function might return rnd_int+1 and still pass this assert
 
 
 # TODO Add a function and at least 3 tests
+def find_reciprocal(x: Union[int, float]):
```

Comment:
Like the type annotation, pay attention to return type
```suggestion
def find_reciprocal(x: Union[int, float]) -> float:
```